code/game/machinery/alarm.dm

```diff
@@ -85,6 +85,19 @@
 
 	var/report_danger_level = 1
 
+/obj/machinery/firealarm/examine(mob/user)
+	. = ..()
+	if(security_level == SEC_LEVEL_GREEN)
+		to_chat(user, "The current alert level is green.")
+	if(security_level == SEC_LEVEL_YELLOW)
+		to_chat(user, "The current alert level is yellow.")
+	if(security_level == SEC_LEVEL_BLUE)
+		to_chat(user, "The current alert level is blue.")
+	if(security_level == SEC_LEVEL_RED)
+		to_chat(user, "The current alert level is red!")
+	if(security_level == SEC_LEVEL_DELTA)
+		to_chat(user, "The current alert level is delta! Evacuate!")
```

Comment:
This can all be simplified by using `The current alert level is [get_security_level()].` It would also be worth it, like geeves said, to put this in a SPAN_NOTICE() macro.